tabpy-tools/tabpy_tools/rest_client.py

```diff
@@ -146,21 +134,19 @@ def query(self, name, *args, **kwargs):
         """Performs a query. Either specify *args or **kwargs, not both.
         Respects query_timeout."""
         if args and kwargs:
-            raise ValueError('Mixing of keyword arguments and positional arguments '
-                                'when querying an endpoint is not supported.')
-        return self.service_client.POST('query/'+name,
-            data={'data':args or kwargs},
-            timeout=self.query_timeout)
-
+            raise ValueError(
+                'Mixing of keyword arguments and positional arguments when '
+                'querying an endpoint is not supported.')
+        return self.service_client.POST('query/' + name,
+                                        data={'data': args or kwargs},
```

Comment:
This looks kind of wonky; wouldn't be surprised if it fails when kwargs gets passed but orthogonal to change here